features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/messagecomposer/MessageComposerPresenter.kt

```diff
@@ -585,10 +585,20 @@ class MessageComposerPresenter @Inject constructor(
                 content = htmlText ?: markdownText
             )
             is ComposerDraftType.Reply -> {
-                messageComposerContext.composerMode = MessageComposerMode.Reply(InReplyToDetails.Loading(draftType.eventId))
+                messageComposerContext.composerMode = MessageComposerMode.Reply(
+                    replyToDetails = InReplyToDetails.Loading(draftType.eventId),
+                    // I guess it's fine to always render the image when restoring a draft
+                    hideImage = false
+                )
                 timelineController.invokeOnCurrentTimeline {
                     val replyToDetails = loadReplyDetails(draftType.eventId).map(permalinkParser)
-                    run { messageComposerContext.composerMode = MessageComposerMode.Reply(replyToDetails) }
+                    run {
```

Comment:
Is this `run` needed?

Reply:
Was needed, but not anymore with 6ee0dd32c32f49f1f3fcbc3277efbc01f6fc9d4f